app/views/posts/index.html.erb

```diff
@@ -1 +1,35 @@
-<p> here you can find all posts </p>
\ No newline at end of file
+<main>
+    <div class="user-container">
+        <section class="user">
+            <img src=<%= @user.photo %> alt="User Photo" class="user-photo">
+            <div class="user-details">
+                <h2 class="user-name"><%= @user.name%></h2>
+                <h5 class="user-post-count">Number of posts: <%= @user.post_counter %> </h5>
+            </div>
+        </section>
+        <section class="user-posts">
+            <h3>All Posts from <%= @user.name %></h3>
+            <% @user.posts.each do |post| %>
+            <a href="/users/<%=@user.id%>/posts/<%=post.id%>" class="user-post-link">
+                <div class="post">
+                    <h4><%= post.title %></h4>
+                    <p><%= truncate(post.text, length: 180) %></p>
+                    <p class="comment-like-count">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+                </div>
+                <div class="comments">
+                    <h4>Comments</h4>
+                    <% post.five_latest_comments.each do |comment| %>
```

Comment:
- [ ] The view cannot access your `five_latest_comments` in your Post model. Kindly make it public.

  ![image](https://github.com/AlaaAlsalem/blogaapp/assets/81468675/c00864fd-32e2-49ae-8560-d8e30858a345)